packages/select/src/MenuButton.tsx

```diff
@@ -105,9 +114,44 @@ const MenuButton = React.forwardRef<HTMLElement, Props>(function MenuButton(
             font-size: ${mobileSizeSet.text}px;
           }
         `,
+        {
+          [css`
+            border-color: ${errorColor};
+            box-shadow: 0px 1px 2px rgba(87, 11, 8, 0.3);
+
+            &:hover,
+            &:active {
+              border-color: ${errorColor};
+              box-shadow: ${mode === Mode.Light
+                ? `0px 4px 4px rgba(87, 11, 8, 0.3),
+              0px 0px 0px 3px ${uiColors.red.light3}`
+                : `0px 4px 4px rgba(87, 11, 8, 0.3), 0px 0px 0px 3px ${uiColors.red.dark2}`};
```

Comment:
We should use `polished.transparentize` (or `.opacify`) with `uiColors` (if it exists), instead of hard coding an `rgba` value

Reply:
As discussed this will be updated during the reskin